### ISSUE TITLE
OP Items

### ISSUE THREAD

**Issue by white-haired-uncle** [Dugy/Legend_of_the_Invincibles#755]
Breaking out discussion which began #715 on items that are perhaps way too powerful and "need" to be broken up, nerfed, made into super items, etc.

Adding Holy Sword.  Other than the fact that 90% of all units have Legacy of Light so some of this is redundant, wow.  For example, illuminates and align=lawful together seems about right for a single item, perhaps with a low level resistance bump or something, but in this case they are just a small part.

**Comment by Dugy** [comment]
Yeah, I can agree that it has a bunch of abilities that are all quite powerful even individually. Does it feel so powerful that it needs to be weakened, either turned into an item set or just by removing some properties, or it could just be made super rare?

>  Other than the fact that 90% of all units have Legacy of Light 

What do you mean by that? Is the random assignment of legacies broken and Legacy of Light is the most common one?

**Comment by Dugy** [comment]
I changed murderlust to work only on offence. The old murderlust is now called _supreme murderlust_ and is obtainable only as AMLA (Werewolf Rider and Abomination).

**Comment by Dugy** [comment]
From your descriptions, it sounds like if Legacy of Light was making lawful units more powerful overall than chaotic units, which could be the root of the problem. It seems really powerful - exceptional illumination + lawful + healing. Maybe some of these upgrades could be removed, like the exceptional illumination.

Holy Sword could have some of its properties removed, I would start with those that are easily available from other sources.

I will think about the rest of things you've said, I have to dash to work now.

**Comment by white-haired-uncle** [comment]
> From your descriptions, it sounds like if Legacy of Light was making lawful units more powerful overall than chaotic units, which could be the root of the problem. It seems really powerful - exceptional illumination + lawful + healing. Maybe some of these upgrades could be removed, like the exceptional illumination.

I suspect a few causes for the light/dark balance thing.

1. Personal preference.  I like Prophets and CMs.
2. I don't think there's really a counter to LoL.  Undead comes to mind, but it seems weak to me and with all of its negative resistance/HP I never really use it
3. Items.  In particular, Purity seems to be a lot cheaper (common) to craft than Deflector of Light. 
4. Enemies seem to tend more toward chaotic.  Illumination is a weapon against them.
5. ToD.  I bet if you were to count all the turns spent in day/night, between underground and long dark/etc, it would come out with a lot more dark.  This might seem like a reason to focus on chaotic units, but given all the other factors it's probably more of a reason to focus on a lot of illumination.
6. Units.  Seems like there are more lawful units available to recruit than chaotic, and the balance is almost certainly heavier on lawful early which means that they have a lot longer to advance.  And undead are mostly pretty weak/fragile, at least at first, so they're harder to level.
7. There was a 7.  I was just thinking of it.

Coming back to items, Hatred and Chaos just seem like they should be tailored for chaotic units based on the names.  And I think one could argue that poison is kind of a "bad" thing to do, in the same vein as necromancy, so probably Intoxicator.

**Comment by Dugy** [comment]
Celestial Messenger and Prophet are generally highly popular units that are probably OP by themselves. I wanted to make them slightly weaker, most likely just by reducing some numbers. Celestial Messenger was recently made even better by making conviction more important.

I could revise the sources of illumination, there really may be many of them and giving stronger versions of illumination. Meanwhile, it's true that darkens usually makes enemies stronger (except for demons). Maybe they could be paired with _despair_?

I am aware of the unpopularity of undead, I even have a concrete list of extra abilities they could get to be more attractive. Previously, I added a few new undead unit types one Halloween a few years ago, but it didn't do much.

The item _Chaos_ intentionally has a bunch of unrelated mods. I created it out of curiosity if it will be useful somewhere, but it seems that its +1 attacks is the main reason to use it. Or maybe skirmisher.

**Comment by Dugy** [comment]
I have changed Holy Sword in 836f8c8.

**Comment by white-haired-uncle** [comment]
> Celestial Messenger and Prophet are generally highly popular units that are probably OP by themselves. I wanted to make them slightly weaker, most likely just by reducing some numbers. Celestial Messenger was recently made even better by making conviction more important.

They are way OP, IMO.  One easy thing I was thinking about was changing the AMLA order so you couldn't take the great stuff so early.

> 
> I am aware of the unpopularity of undead, I even have a concrete list of extra abilities they could get to be more attractive. Previously, I added a few new undead unit types one Halloween a few years ago, but it didn't do much.

I am quite interested in the undead, probably because I've never done much with them.  I only recently learned about advancing soulless to interesting things like Abomination and Monstrosity (which may need some attention).  I had a thought that might make them even better, soulless advance based on their variation.  So the _only_ way to get a new Wose Lich (whatever) is to advance a soulless with a wose variation.  Nice side effect is that would put a lot more emphasis on plague, so instead of your leaders slaughtering everything in sight you end up using them as support (softening up/slowing enemies, providing leadership, etc).

My problem is not so much the units (though they do need some work), but mainly that when close to half of my army is lawful and I have a lot of illumination, the chaotic units and lawful/illum don't play well and it becomes a bother to keep them separate.  The undead lines are interesting, just too much hassle at the moment IMO.  Perhaps more scenarios that are undead-only/mostly so you _have_ to build some up.

**Comment by Dugy** [comment]
> They are way OP, IMO. One easy thing I was thinking about was changing the AMLA order so you couldn't take the great stuff so early.

Yeah, requiring some boring AMLAs before the exceptional ones could contribute to making them less overpowered.

**Comment by white-haired-uncle** [comment]
Just crafted a Soul Hunter staff for my prophet.  Pretty cheap, a little resistance, waste of good weapon damage, but it has murderlust.  This lead me to wonder if murderlust would not be better as a weapon special than an ability (probably with a different name, murderlust sounds like an ability).  I wouldn't advocate for changing anything now since it was just updated, just a thought I wanted to get down -- maybe it applies to something else.

Another idea to restrict murderlust or something similar is to combine it with an attack type like backstab or charge, maybe even berserk.

**Comment by Dugy** [comment]
I think it would compete too much with drain-like weapon specials if it was a weapon specials, and there are already quite a few drain-like weapon specials that are too similar to each other.

**Comment by white-haired-uncle** [comment]
Brightslash is pretty useful.  Combine it with high HP/resist, and you have a healer who can defend from the front as well.

heals/absorb(2)/damage+20% would be a pretty good item, above average I think.  regenerates(16) is just way too much on top of that IMO.

**Comment by Dugy** [comment]
Maybe I could reduce the absorb to 1, as absorb 2 is a rather strong an rare ability.

**Comment by white-haired-uncle** [comment]
Wrathful Combat Tech.  Furious5 + wrath!

I gave it to a Dwarf with the Blue Orb and Anger.  The axe can get a ton of hits, but the mace has magical and still gets a lot this way.  Either way, he's a mean little fella.  I guess I could craft anger for the axe and use that anger mace too.

This time I'm looking to combine WCT with Blue Orb and Stormrend.  I can't find a unit with inherent marksman for sword (odd), but with 40-50 trickery hits per turn, sooner or later I won't need it.  Good thing the lightning gets overridden by fire.

**Comment by Dugy** [comment]
Furious 5 seems like something that may have an extreme potential for exploitation. Maybe I could remove it for all units maybe except the Berserker line.

**Comment by white-haired-uncle** [comment]
Marrowrend looks a little troubling.  Lot of good stuff, no drawbacks (assuming you're not scared of anger).

The thing that jumps out at me, anger and +30% attacks?

**Comment by white-haired-uncle** [comment]
Warheart depends on Warguard, which means once I have one WG I can craft 3 WH and get +3 attacks.  If WG depended on WH instead it would seriously degrade my ability to abuse things like WCT and Anger.

**Comment by Dugy** [comment]
Marrowrend requires a black pearl to craft, Anger is far cheaper and both are quite strong otherwise. Maybe making Marrowrend a little weaker and changing Anger to something else could help here?

> Warheart depends on Warguard, which means once I have one WG I can craft 3 WH and get +3 attacks. If WG depended on WH instead it would seriously degrade my ability to abuse things like WCT and Anger.

And the increase damage aura from Warheart will stack multiplicatively from the three copies, right? My idea was that stacking it three times would get you needless duplicate properties, but that's not the case, these stack multiplicatively. I could achieve something by requiring some other conflicting item to have the increase damage aura, but moving the set effect to gauntlets should work as well.

**Comment by white-haired-uncle** [comment]
Same issue with Soul Thrash (wrath, non-craftable) and Hatred (+1 attack, craftable).  You may want to consider swapping wrath and +1 attack, since crafting multiple wrath does nothing for you while +1 attack stacks.

Or not, some exploits make things interesting.  But it's worth thinking about to make sure it works the way you intend.

**Comment by Dugy** [comment]
Are you able to get enough Black Pearls for having multiple instances of Hatred?

**Comment by white-haired-uncle** [comment]
Yeah.  My Predator has two and I have a BP left.  That's playing with reduced drops, particularly in ch5.

Not sure if you can buy them, but if so I could have picked up one or two more in Bitter Swamps, where I always have way too much gold.

**Comment by Dugy** [comment]
A black pearl should allow you to make something very nice. Your supply of black pearls is clearly limited, you can equip one unit with them and then you're out. Or is Hatred the obvious choice of the ones you can make with them?

I am a little surprised you are using a Predator, my notes say that Predator isn't a very popular choice to develop.

**Comment by white-haired-uncle** [comment]
I had a loyal[1] Poacher, so I advanced him. And really, he's not a bad unit.  So far I've only take Ice Dragon AMLAs.  With nightstalk and his 38x6 + lurk bow he'd make a fine assassin here in the tunnels.  But he kills pretty much anything in one turn anyway.

![pred1](https://github.com/Dugy/Legend_of_the_Invincibles/assets/135195815/9ca469e2-0135-436e-b649-f72b9e01899e)
![pred2](https://github.com/Dugy/Legend_of_the_Invincibles/assets/135195815/411e0132-1404-4ae6-ab45-13cdba53c8fe)

Chaos, Hatred, and Marrowrend are pretty much all I use BPs for.  I probably already had Soul Thrash (and daggers usually suck, so this is/was a good choice for the damage penalty) and/or I had the other gems to go Hatred vs Chaos.  [also, I haven't synced in at least a week, so I may be making choices based on older weapons/specials).

[1] In ch5, "loyal" is basically worthless.  Need to fix that somehow.  I have some "interesting" ideas.

**Comment by Dugy** [comment]
Okay, so it looks like Predator doesn't deserve being as underused as he is now.

> Chaos, Hatred, and Marrowrend are pretty much all I use BPs for.

Chaos and Hatred are the only two crafts that take 1 black pearl. So it looks they are worth their cost. On the other hand, Marrowrend seems to be much better than the other weapon crafts for 1 black pearl. Maybe it's time to nerf it a little?

> In ch5, "loyal" is basically worthless. Need to fix that somehow. I have some "interesting" ideas.

Yeah. What ideas do you have? Something like a free autorecall?

**Comment by white-haired-uncle** [comment]
Violent Staff:  Attached, my elf overlord has taken faster at combat twice, and hold searing ice.  So his violent beatdown gives him 18 attacks (I wonder if I can put furious on that with WCT).

Shoud VB/anger double the bonus from searing ice?  It's only one more attack, in this case, but it makes a point.  Perhaps multiply should come before add (though this may have side effects if applied in the general case with items that give like 30% -- they may never give any additional hits).

Also, with the VB set you get firecast, which according to the description fires with each hit?  So he could potentially get what is effectively an extra 18 attacks per turn?

While we're here, check out the FI next to him, who happens to be holding the full VM set.  She is (will be) an incredible healer.  With conviction(30). Add on the VM set and she can do some pretty serious damage.  Her base HP are pretty decent, add a little resistance and you have a front line unit with massive healing and conviction.  The one in the attached has only taken two AMLAs and she's already something of a brute. One thing I would consider is making her healing grow more slowly (like +4/AMLA) and her conviction much more slowly (like 5%/AMLA -- currently she needs just two AMLA to get 30!).


[LotI1-Southern Guardian Room-Auto-Save11.gz](https://github.com/Dugy/Legend_of_the_Invincibles/files/15076472/LotI1-Southern.Guardian.Room-Auto-Save11.gz)

**Comment by Dugy** [comment]
> I would start with the +30% attacks.

Yeah.

> the human intelligence factor makes 2x att/def a huge bonus (e.g. we humans can pick which unit/attack it applies to and which unit(s) to use it against).

The AI can do that too.

> A more logical scheme would be that you get whatever gold the enemy has left when you defeat them, adjusted for the spoils of victory. And supporting your army should be a constant net drain so that's encouragement to finish early as well.

In a modern war, soldiers engaged in fighting take much more resources than soldiers who are just sitting around. Ammunition, repairing equipment, replacing short-lived components and such things are far more expensive than soldiers' wages. I assume it was similar in times of yore, swords and armour also needed repair.

So if soldiers are not geared for battle, the resources they take are much smaller.

> A particular problem with ch5 is XP farming. Once the enemy leaders are taken care of, you can pretty much hang out at the spawn points and pick off individual units forever.

Maybe resume the upkeep once the objective is complete?

> Upkeep also provides an encouragement to utilize certain units, like undead (who also should have cheaper upkeep since they have more available food sources / less food need). 

You need to keep buying the Black Essence that keeps them in your control.

> Shoud VB/anger double the bonus from searing ice? It's only one more attack, in this case, but it makes a point. Perhaps multiply should come before add (though this may have side effects if applied in the general case with items that give like 30% -- they may never give any additional hits).

These two things are stackable because of the way how the game works. The bonuses from items, like The Searing Ice, apply to the unit's attacks. The weapon special modifies it on the fly if given conditions apply.

If you think The Searing Ice is too powerful, it's possible to make it weaker.

> Add on the VM set and she can do some pretty serious damage. 

Violent Mage's set is meant to give good melee bonuses to mages, without doing much to their magical abilities. It's coincidentally useful to Faerie Incarnation because she doesn't have much damage with magical attacks in the first place.

But Faerie Incarnation is overused for rather obvious reasons and is on my list of units that need to be made weaker.

**Comment by white-haired-uncle** [comment]
> > the human intelligence factor makes 2x att/def a huge bonus (e.g. we humans can pick which unit/attack it applies to and which unit(s) to use it against).
> 
> The AI can do that too.

The AI gets an anger special where we decide, if it gets one at all.  The player can do things like craft Anger for a scythe to be used with whirlwind, and only for the unit that has the Blue orb and Warheart/guard.  So while it seems like anger (on a player weapon) applies to AI and human equally since it doubles the AIs defensive attacks as well, I'm convinced the extra AI attacks don't come anywhere near offsetting the advantage to the human.

> 
> > A more logical scheme would be that you get whatever gold the enemy has left when you defeat them, adjusted for the spoils of victory. And supporting your army should be a constant net drain so that's encouragement to finish early as well.
> 
> In a modern war, soldiers engaged in fighting take much more resources than soldiers who are just sitting around. Ammunition, repairing equipment, replacing short-lived components and such things are far more expensive than soldiers' wages. I assume it was similar in times of yore, swords and armour also needed repair.
> 
> So if soldiers are not geared for battle, the resources they take are much smaller.

Smaller, maybe, but positive.  Wesnoth treats them as free, which is all kinds of wrong.  [I also don't like things like if I choose not to recall my mounted units on a mountainous map they can't cross, somehow they're magically available when we get to the other side].

As far as real world, I suspect you're way off.  Setting aside multimillion dollar cruise missles and the like and looking only at person weaponry and armor, I'm sure food/clothing/shelter costs are vastly higher per soldier than gear.  And that should hold even if you only consider days in actual combat.  It was almost certainly more so in the old days, when "ammunition" was far less expendable/expensive and troops were more expendable.

> 
> > A particular problem with ch5 is XP farming. Once the enemy leaders are taken care of, you can pretty much hang out at the spawn points and pick off individual units forever.
> 
> Maybe resume the upkeep once the objective is complete?

I think that might be the answer right there.  Do you happen to know where the DISABLE_UPKEEP changes get reverted?

> If you think The Searing Ice is too powerful, it's possible to make it weaker.

It was just an example, the unit in question happened to be carrying it, but it is one of the more powerful weapons.  I think it's a close second to Doombringer simply because of the varying damage types.

> 
> > Add on the VM set and she can do some pretty serious damage.
> 
> Violent Mage's set is meant to give good melee bonuses to mages, without doing much to their magical abilities. It's coincidentally useful to Faerie Incarnation because she doesn't have much damage with magical attacks in the first place.
> 
> But Faerie Incarnation is overused for rather obvious reasons and is on my list of units that need to be made weaker.

I really only use Elder Mages (white become CMs), and they are already crazy powerful.  Not having a powerful melee is almost necessary for them (and they do have a doom attack or something like that available).

And the +25% spell bonus with cloak and armor is certainly not chicken feed.

**Comment by white-haired-uncle** [comment]
Just figured out that Violent Mage works pretty well on a lich.  Actually just crafting Anger for his touch attack works, but in my case the VB does about 50% more damage.  This would be extremely cool if all my enemies weren't mechanical (undrainable).

I'm thinking the VM set, and then either Konrad's Might for marksman or Widowmaker for charge.   I tried Anger + VB, but anger doesn't stack with itself. 50-60x12 with drains?  Maybe Stormgrip for the lightning damage type. 

If I create it from ToL, I should be able to pick up a spear (Corpsegrinder for evisceration of course) and a sword (something boring like Cuncators or Dominion for the resistances, murderlust is probably a waste, retribution would just be mean)

**Comment by Dugy** [comment]
> The AI gets an anger special where we decide, if it gets one at all.

Aha, that is what you meant. I don't argue that.

What I meant is that AI can also choose which unit will it use anger against.

> Wesnoth treats them as free, which is all kinds of wrong.

They are significantly cheaper than battle ready units. Or they are around on their own expense for the case you wanted to fight with them again.

> Setting aside multimillion dollar cruise missles and the like and looking only at person weaponry and armor, I'm sure food/clothing/shelter costs are vastly higher per soldier than gear.

Rifles and body armour are cheap, but tanks, IFVs and artillery aren't. Military vehicles are not designed to be able to survive a lot of kilometres, because they take much more damage in combat. They're not like civilian cars that can travel 100 000s of km only with basic maintenance. And ammunition for machine guns and artillery costs a lot, because typical tactics expect using large amounts of ammunition. Dropping lots of bombs and shooting lots of bullets anywhere the where enemies could be is a good tactic to avoid losses. Waiting is so much cheaper.

> It was almost certainly more so in the old days, when "ammunition" was far less expendable/expensive and troops were more expendable.

Ammunition was less plentiful, but armour also needed fixing after battles, swords rusted if not properly stored, shields were often designed for enemy weapons to get stuck in them and were not really reusable.

Troops were maybe even less expendable, because it took years to master archery (in fact, firearms were initially spread not because they were better than arrows, but because they took much less training), swords are also very counter-intuitive to use. There were times where armies didn't engage because it wasn't worth the damage, they chose instead to attack villages instead to prevent the enemy to keep the army funded (the strategy was called _chávauchéé_).

> I think that might be the answer right there. Do you happen to know where the DISABLE_UPKEEP changes get reverted?

If I recall correctly, the events it places have ids and you can use those ids to remove those events.

> It was just an example, the unit in question happened to be carrying it, but it is one of the more powerful weapons. I think it's a close second to Doombringer simply because of the varying damage types.

I think I underestimated how good it is. I wanted to make it rather good because I spent the effort to make all those pictures, but I overdid it.

Maybe I could remove the _suck_ property or something.

> Not having a powerful melee is almost necessary for them (and they do have a doom attack or something like that available).

I don't understand what do you mean by this.

> And the +25% spell bonus with cloak and armor is certainly not chicken feed.

I forgot about that. I suppose I could remove some of it.

> Just figured out that Violent Mage works pretty well on a lich.

This seems like something very exploitable. On the other hand, the Lich setup takes so much preparation that it almost looks well deserved.

**Comment by white-haired-uncle** [comment]
> > Not having a powerful melee is almost necessary for them (and they do have a doom attack or something like that available).
> 
> I don't understand what do you mean by this.

The Elder Mage is so powerful that I usually only bother creating one of them.  Once reasonably well advanced (half a dozen or so AMLAs) and geared, the explosive fire kills everything in sight.  Not having much of a melee attack (to use in defense) means you have to pay at least a little attention to putting them in danger.  Here's an example.  I got lucky with Fire Dragon legacy, then added Shako and most of the Unthought set, giving huge fire penetration and resistance (to nullify reflect).  While the damage doesn't look all that impressive, for some reason his explosive fire breath (which I'm pretty sure is just an alternative to advancing his fire blast, I know my elder mages always have a great explosive fire attack) normally kills several units every time it's used.

I didn't think explosive was supposed to "travel" more than one tile until it became a lot more powerful than just 37 damage (even with the bump from penetration), but it seems to.

![elder](https://github.com/Dugy/Legend_of_the_Invincibles/assets/135195815/8b9bd684-7c08-4fff-94ff-00febda9186c)


> > Just figured out that Violent Mage works pretty well on a lich.
> 
> This seems like something very exploitable. On the other hand, the Lich setup takes so much preparation that it almost looks well deserved.

You get one from Journey South (I think that's it).  Aside from that I think it's ToL, which I've never really played with because I'd want to put so much thought into it.  And I'm pretty sure I'd want to kill off a prophet (for the sword and spear) to get it, and while I'm a huge fan of the lich, losing a prophet is hard for me to do.

I think you could create an nearly invincible lich, and the VM set would be a key piece, but I don't really see a problem.  As you say, it's well deserved, and rare, and if the player feels it makes the game too easy they could always just not do it, or kill it and try to make an even better one.

**Comment by Dugy** [comment]
Elder Mage might be one of the units whose numbers need to be reduced a little. He had to be strong because he's a level 5 unit after all, but it was a little overdone. But, on the other hand, he has a rather obvious flaw, the vulnerability in melee.

The cutoff for larger area of explosive damage is 24 if I understand the code correctly, not 37.

**Comment by white-haired-uncle** [comment]
I don't find the weakness in melee to be a flaw, or even a weakness really.  A not so obvious "strength" of melee weakness is it allows taking items with huge melee damage specials without incurring much of a penalty.  For example, if I knew that explosive hits contributed to wrath I would seriously consider giving him Soul Thrash.  

I was thinking his doom attack was melee, but it's ranged, which is probably why I never use it.  But if it was melee, I'd be more inclined to take it because then by using it with its pathetic damage I'm not giving up as much as I would be if my normal melee damage was much greater. And I say that knowing how completely illogical it is.

BTW, for Anger/Marrowrend, I was thinking lose the +30% attacks, and then make the other increase based on difficulty (+100/+75/+50%).  That way you're not taking much away from easy (and not the part that would be really noticable), but they aren't such beasts on hard (but still definitely worth taking/crafting).

Or weaken one, probably Marrowrend, and make the other a rare item (which would both limit its availability and remove the ability to choose the sort via crafting).

**Comment by white-haired-uncle** [comment]
With 100 hits per attack, I probably should drop stormgrip.  Hate to give up lightning damage, but 100 hits with suck!  I have a warheart to use in its place, if I craft 3 warheart for another +3 (x5 x2) attacks...  Too bad I only have on potion of speed.

And she doesn't even have the blue orb this time.

![manta](https://github.com/Dugy/Legend_of_the_Invincibles/assets/135195815/c01c896e-c5f7-41ff-a30a-aaaa630cdefc)

**Comment by Dugy** [comment]
So do we change Marrowrend's attack count from 30% to -30%?

**Comment by white-haired-uncle** [comment]
I would or maybe {30,0, -30}.

I was also wondering if potion of speed (for example) should stack on itself, or perhaps if you should be able to have more than one potion active at a time.  But that might be sucking the fun out of it.  Other than healing potions, and the potion of authority at a couple key points (beginning of Army is Born, end of Gates of Hell), I never really use potions except when I want to screw around doing something like mixing multiple speed potions with anger and WCT, or beating Akula/Uria in one turn.

**Comment by Dugy** [comment]
> I would or maybe {30,0, -30}.

Done.

> I was also wondering if potion of speed (for example) should stack on itself [...]

To be honest, I never thought much about the possibility to stack the same potion multiple times. They were meant to help you deal with scenarios that are more difficult than you can handle. But stacking the same potion several times can give a unit a single stat so powerful that it could do what wasn't expected at all, which is not exactly desired.

**Comment by white-haired-uncle** [comment]
Book of Fireballs.  One of my favorites, as it gives a ranged attack to my scythemasters, destroyers, etc.  That's pretty big thing IMO.  Having the very first AMLA not only give a ranged attack, but incinerate?  That's too much.  Incinerate should come later, and perhaps be it's own very weak ranged attack so you have to choose between big damage and incinerate, and on defense you probably won't incinerate very often.

**Comment by Dugy** [comment]
Okay, I moved the incinerate into a bonus attack that is based on fireball but has -40% damage.

**Comment by nerffrog** [comment]
While I agree that items need rebalancing, I believe that most imbalance rather comes from stupidly strong units. To put things into perspective, I managed to beat part 2 of the campaign in an itemless run. Granted it was on easy mode, the fact that you can kill Lilith, or Abaddon, or Uria with just a duke, some faerie incarnations, some celestial messengers, and 2 snow hunters is ridiculous. Obviously their stats on their own are not what made it possible, but rather dragon legacies. It is not difficult at all to get a dragon legacy on something as easy to level up as a snow hunter, and pair it with a conviction (40) FI or conviction (33) CM. I was easily dealing 50*6 explosive damage with a single snow hunter, sometimes up to 71*6 (this much damage is dealt without items and after adjusting for enemy resistances).
FI clearly needs a nerf to its conviction and possibly her health as well. Preventing resistance penetration from affecting all enemies was a step in the right direction, but I think penetration provided by dragon legacies should be reduced from 21% to 10-15%. There's absolutely no reason for a 21% penetration on an attack that deals 19-6 base damage. Also, it is possible to stack Legacy of Kings with abilities like Legacy of Kings (15) or whatever it was, as they're treated differently, might be worth checking that out.
I know this thread is for discussing items, I just wanted to point out that defeating Uria itemless was possible, which says a lot about the state of units themselves.

**Comment by white-haired-uncle** [comment]
>  conviction (40) FI or conviction (33) CM. I was easily dealing 50-6 _explosive_ damage with a single snow hunter, sometimes up to 71-6 (this much damage is dealt without items and after adjusting for enemy resistances). FI clearly needs a nerf to her conviction and possibly her health as well. Preventing resistance penetration from affecting all enemies was a step in the right direction, but I think penetration provided by dragon legacies should be reduced from 21% to 10-15%. 

FWIW, conviction for a CM currently tops out at 20 (probably only in beta), and you now have to take a few other AMLAs before you can start down that path.

I raised the same case about FI the other day, in particular with the Violent Mage set (awesome healers shouldn't have great damage and high/cheap conviction, IMO).

**Comment by Dugy** [comment]
So not only Legacy of Light is OP, the dragon legacies could also be a little weakened.

Celestial Messenger and Faerie Incarnation are both on my list of units that need to be made weaker.

**Comment by white-haired-uncle** [comment]
Mystic Armor of the Untouchables.  It's basically Godless, but with suck(1) and 15% defense.  Even if the defense bonus will eventually be negated by every enemy having magical attacks, that's really powerful.  There are other armors not nearly as powerful that have -2 or even -3MP.  And based on the name, I wouldn't be surprised if it had antisocial or something similar.

**Comment by Dugy** [comment]
Yeah, definitely overpowered. I will give it the _antisocial_ ability as you suggested.

**Comment by arobinson** [comment]
Even though it is fun, I find the potions are the most OP items. If you simply make sure to never level up on offense, potions become permanent. This means madman potion, speed potion, mundane potion, titanic strength, etc can be used throughout the entire game with one potion.  For example, the unholy hunger potion (or whatever it's name is) has allowed my Efraim to have 1767 HP and growing.

<img width="197" alt="image" src="https://github.com/Dugy/Legend_of_the_Invincibles/assets/93213/7c7d9245-e250-40e5-a54e-7bb325339734">

**Comment by Dugy** [comment]
That is a bug. They were meant to be removed on levelup, even if it's during the enemy turn.

**Comment by white-haired-uncle** [comment]
That would be great for chapter 10.  I always load the kids up with potions near the end of Gates of Hell.  They lose their advances when you get to ch10, but the effects of potions remain.  I wish I'd known not to level them up, though it's difficult to get through those first few scenarios without doing so (I've actually toyed with the idea of a "smash all items in inventory" option so you could carry a ton of gems over to help gear your units in ch10).

And potions were a great exploit with doppelgangers, as the effects are cloned but not as potions so they don't go away on advancement.  Of course, now they don't recall, it's just a clever trick.
